lib/power_action_utils.pm

```diff
@@ -272,7 +263,14 @@ sub power_action {
                 reboot_x11;
             }
             elsif ($action eq 'poweroff') {
-                poweroff_x11;
+                # Temporary workaround, because shutdown module is marked as
+                # 'cancelled' on s390x backend when shutting down from desktop.
```

Comment:
Could you please add reference to progress ticket? Maybe with soft-failure. Other than that LGTM.

Reply:
Added.